charts/cytomine/templates/pims/pims.yaml

```diff
@@ -0,0 +1,85 @@
+{{ if .Values.pims.deploy }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "cytomine.name" . }}-pims
+  labels:
+    role: pims
+    app: {{ include "cytomine.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: {{ .Release.Name }}-pims
+    release: {{ .Release.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "cytomine.name" . }}-pims
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        role: pims
+        app: {{ template "cytomine.name" . }}-pims
+        release: {{ .Release.Name }}
+    spec:
+      serviceAccountName: {{ include "cytomine.serviceAccountName" . }}
+      {{- with .Values.global.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      volumes:
+        - name: images
+          persistentVolumeClaim:
+            claimName: {{ template "cytomine.name" . }}-image-claim
+        - name: image-temp
+          persistentVolumeClaim:
+            claimName: {{ template "cytomine.name" . }}-temp-image-claim
+        - name: pims-logging
+          configMap:
+            name: {{ template "cytomine.name" . }}-pims-logging
+      containers:
+        - name: {{ .Chart.Name }}-pims
+          image: "{{ .Values.images.pims }}"
+          imagePullPolicy: {{ .Values.images.pullPolicy }}
+          envFrom:
+            - configMapRef:
+                name: cytomine-pims-config
+          env:
+            - name: POSTGRES_USER
```

Comment:
Should provide `POSTGRES_DB` as well? Or does the software not allow configuring the database?

Reply:
It allows configuring the database. Added it now.